interface/forms/gad7/gad7.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * gad-7.inc - common includes and constants for the gad-7 form
+ * version 1.0.0  July 2020
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    ruth moulton
+ * @copyright Copyright (c) 2021 ruth moulton ruth@muswell.me.uk
```

Comment:
Use this format for your author and copyright tags in this file and other files:
```
 * @author      Ruth Moulton <moulton ruth@muswell.me.uk>
 * @copyright Copyright (c) 2021 Ruth Moulton <ruth@muswell.me.uk>
```